<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>presetDownloadW</class>
 <widget class="QWidget" name="presetDownloadW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>423</width>
     <height>363</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frameContent">
      <property name="minimumSize">
       <size>
        <width>421</width>
        <height>361</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
        #frameContent{
            background: white;
            border: 1px solid #ddd;
            border-radius: 12px;
        }
       </string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>8</number>
       </property>
       <item alignment="Qt::AlignHCenter|Qt::AlignTop">
        <widget class="QLabel" name="lblImage">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>Images/preset_download_icon.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter|Qt::AlignTop">
        <widget class="QLabel" name="lblTitle">
         <property name="maximumSize">
          <size>
           <width>600</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Preset Download</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter|Qt::AlignTop">
        <widget class="QLabel" name="lblSubtitle">
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Browse &amp; Download</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnPresetDownload">
         <property name="minimumSize">
          <size>
           <width>389</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>389</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">text-align: left;</string>
         </property>
         <property name="text">
          <string>Download Preset</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCreator">
         <property name="minimumSize">
          <size>
           <width>389</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>389</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">text-align: left;</string>
         </property>
         <property name="text">
          <string>More From This Creator</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnReport">
         <property name="minimumSize">
          <size>
           <width>389</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>389</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">text-align: left;</string>
         </property>
         <property name="text">
          <string>Report</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::HLine</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignLeft">
        <widget class="QPushButton" name="btnBackToPreset">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">
            #btnBackToPreset {
                border: none;
                background: transparent;
                text-align: left;
                padding: 6px 0px;
            }
            #btnBackToPreset:hover {
                text-decoration: underline;
            }
          </string>
         </property>
         <property name="text">
          <string>‚Üê Back to Preset Menu</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
